src/UI/RowList.elm

```diff
@@ -0,0 +1,469 @@
+module UI.RowList exposing
```

Comment:
So, I'm not very sure about this name 🤔  I think some frameworks would call this ListView. React-native that calls `FlatList` and I kinda like this name, WDYT?

Reply:
I just realize that the lists are merged here, so maybe just call ListView would be better, it is more "generic", not sure tho, up to you